src/EditorFeatures/Core/Implementation/Diagnostics/DiagnosticsClassificationTaggerProvider.cs

```diff
@@ -17,8 +17,7 @@
 namespace Microsoft.CodeAnalysis.Editor.Implementation.Diagnostics
 {
     [Export(typeof(ITaggerProvider))]
-    [ContentType(ContentTypeNames.CSharpContentType)]
-    [ContentType(ContentTypeNames.VisualBasicContentType)]
+    [ContentType(ContentTypeNames.RoslynContentType)]
```

Comment:
Yes.  And this shoudl be safe to switch over.  This only affects diagnostics that are created with teh unnecessary-tag attached to them.  TS does not create these (current) so it won't affect them (@paulvanbrenk @jramsay to confirm).   If they do create them in the future, they'll have the ability to have faded out text like C# and VB use for unnecessary code.